pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -111,6 +120,30 @@ func (pc *PackageControllerClient) InstallController(ctx context.Context) error
 	return nil
 }
 
+// InstallPBCResources installs Curated Packages Bundle Controller Custom Resource
+// This method is used only for Workload clusters
+// Please refer to this documentation: https://github.com/aws/eks-anywhere-packages/blob/main/docs/design/remote-management.md
+func (pc *PackageControllerClient) InstallPBCResources(ctx context.Context) error {
+	templateValues := map[string]string{
+		"clusterName": pc.clusterName,
+		"namespace":   constants.EksaPackagesName,
+	}
+
+	result, err := templater.Execute(packageBundleContrllerYaml, templateValues)
```

Comment:
Instead of using go templates, can we build the go struct and marshal it into yaml?
Makes the code easier to read and to extend :)

Reply:
I think that would be an interesting thought but in this case both formats should be readable and I could argue that the go templates in this case is more applicable and makes it easy to read. For extendability, if there is a need then yes we can make the change but in this case I think the templating should work well.

Reply:
Ok if we are trying to get away from that, we also have awssecret.yaml which is in the same line as this file. Let me revisit both in a later PR.